rocketpy/mathutils/function.py

```diff
@@ -110,20 +110,19 @@ def __init__(
         (II) Fields in CSV files may be enclosed in double quotes. If fields
         are not quoted, double quotes should not appear inside them.
         """
-        if inputs is None:
-            inputs = ["Scalar"]
-        if outputs is None:
-            outputs = ["Scalar"]
 
-        inputs, outputs, interpolation, extrapolation = self._check_user_input(
-            source, inputs, outputs, interpolation, extrapolation
-        )
+        inputs, outputs = Function._validate_inputs_outputs(inputs, outputs)
 
         # initialize variables to avoid errors when being called by other methods
         self.get_value_opt = None
         self.__polynomial_coefficients__ = None
         self.__akima_coefficients__ = None
         self.__spline_coefficients__ = None
+        self.EXTRAPOLATION_TYPES = {
+            "zero": 0,
+            "natural": 1,
+            "constant": 2,
+        }
```

Comment:
In this case I preferred to save it in the class because it only makes sense for the Function class, cannot be used out of it as a global variable.

Honestly, it is not necessary and I don't like it. I think it was used to compare `something == 1` instead of `something == "some_string"`, in a desperate trial of speeding up the code.